### ISSUE TITLE
Support pageSize in c8y api command

### ISSUE THREAD

**Issue by k-butz** [reubenmiller/go-c8y-cli#222]
It seems the `--pageSize` option is not passed when using the `c8y api`command. Here's a sample passing pageSize 10 (not added in the URL sent to c8y):

```sh
-> c8y api "/service/advanced-software-mgmt/software?deviceId=25458010" --pageSize 10 -v
2023-03-04T13:59:55.485+0100	INFO	Binding authorization environment variables
2023-03-04T13:59:55.486+0100	INFO	activityLog path: /Users/kobu/.cumulocity/activitylog/c8y.activitylog.2023-03-04.json
2023-03-04T13:59:55.486+0100	INFO	Loaded session: /Users/kobu/.cumulocity/session.json
2023-03-04T13:59:55.486+0100	INFO	command: c8y api /service/advanced-software-mgmt/software?deviceId=25458010 --pageSize 10 -v
2023-03-04T13:59:55.488+0100	INFO	Max jobs: 0
2023-03-04T13:59:55.489+0100	INFO	worker 1: started job 1
2023-03-04T13:59:55.489+0100	INFO	Headers: map[Accept:[application/json] Authorization:[Bearer {token}] User-Agent:[go-client] X-Application:[go-client]]
2023-03-04T13:59:55.489+0100	INFO	Sending request: GET https://{host}/service/advanced-software-mgmt/software?deviceId=25458010
2023-03-04T13:59:55.621+0100	INFO	Status code: 200
2023-03-04T13:59:55.621+0100	INFO	Response time: 132ms
2023-03-04T13:59:55.621+0100	INFO	Response Content-Type: application/json
2023-03-04T13:59:55.622+0100	INFO	Response Length: 0.7KB
2023-03-04T13:59:55.622+0100	INFO	Unfiltered array size. len=5
2023-03-04T13:59:55.622+0100	INFO	View mode: auto
2023-03-04T13:59:55.626+0100	INFO	No matching view detected. defaulting to '**'
...etc.
```
It returns 5 elements, 6 are returned when adding pagesize manually to the URL. 

Tested in version 2.20.2.

**Comment by reubenmiller** [comment]
This is actually by design, as I didn't want to assume that the `c8y api` call is related to a collection. As this command can also be used to get a single object (e.g. `c8y api inventory/managedObjects/12345`), so adding a `pageSize` query parameter on those calls does not make too much sense.

So at the moment if you want to specify the page size you can either provide it in the url, or using the `--customQueryParam`.

```sh
c8y api "inventory/managedObjects?pageSize=6"
```

```sh
c8y api "inventory/managedObjects" --customQueryParam "pageSize=6"
```

Though maybe it will be more useful if the global `--pageSize` flag is respected also in `c8y api` considering that pagination seems to work (using `--totalPages` and `--includeAll`).

**Comment by reubenmiller** [comment]
Resolved by #223 and included in v2.21.0
